src/System.Runtime/tests/System/ArgumentExceptionTests.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+using ExceptionUtility = System.Tests.ExceptionUtility;
```

Comment:
Why is this necessary?

Reply:
I have used the pattern used in the existing tests. I can change it to `using System.Tests`